state/state_test.go

```diff
@@ -1043,7 +1042,9 @@ func TestConsensusParamsChangesSaveLoad(t *testing.T) {
 	}
 
 	for _, testCase := range testCases {
-		p, err := stateStore.LoadConsensusParams(testCase.height)
+		pbParams, err := stateStore.LoadConsensusParams(testCase.height)
+		p := types.ConsensusParamsFromProto(pbParams)
+
 		assert.Nil(t, err, fmt.Sprintf("expected no err at height %d", testCase.height))
```

Comment:
can we move this up? right after `pbParams, err :=`

Reply:
moved it into the load function.